games-sports/xmoto/xmoto-0.5.11.recipe

```diff
@@ -9,77 +9,83 @@ COPYRIGHT="2006-2014 Nicolas Adenis-Lamarre
 	2009-2010 Jens Erler
 	2005-2006 Rasmus Neckelmann"
 LICENSE="GNU GPL v2"
-REVISION="2"
+REVISION="3"
 SOURCE_URI="http://download.tuxfamily.org/xmoto/xmoto/0.5.11/xmoto-0.5.11-src.tar.gz"
 CHECKSUM_SHA256="a584a6f9292b184686b72c78f16de4b82d5c5b72ad89e41912ff50d03eca26b2"
-PATCHES="xmoto_x86-0.5.11.patchset"
+if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+	PATCHES="xmoto-$portVersion-gcc2.patchset"
+else
+	PATCHES="xmoto-$portVersion.patchset"
+fi
 
-ARCHITECTURES="x86_gcc2"
-SECONDARY_ARCHITECTURES="x86"
+ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
 
 PROVIDES="
 	xmoto$secondaryArchSuffix = $portVersion
 	cmd:xmoto$secondaryArchSuffix = $portVersion
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	mesa_swrast
+	mesa${secondaryArchSuffix}_swpipe
 	lib:libbz2$secondaryArchSuffix
 	lib:libcurl$secondaryArchSuffix
-	lib:libgl$secondaryArchSuffix
-	lib:libglu$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
 	lib:libintl$secondaryArchSuffix
 	lib:libjpeg$secondaryArchSuffix
 	lib:liblua$secondaryArchSuffix
 	lib:libode$secondaryArchSuffix
-	lib:libpng$secondaryArchSuffix
-	lib:libsdl$secondaryArchSuffix
-	lib:libsdl_mixer$secondaryArchSuffix
-	lib:libsdl_net$secondaryArchSuffix
-	lib:libsdl_ttf$secondaryArchSuffix
+	lib:libpng16$secondaryArchSuffix
+	lib:libSDL_1.2$secondaryArchSuffix
+	lib:libSDL_mixer_1.2$secondaryArchSuffix
+	lib:libSDL_net_1.2$secondaryArchSuffix
+	lib:libSDL_ttf_2.0$secondaryArchSuffix
```

Comment:
```suggestion
	lib:libSDL_ttf_2.0$secondaryArchSuffix
```

Shouldn't this be 1.2 or removed? We can't mix different version of sdl like that...

Reply:
SDL ttf up to 2.0.11 goes with SDL 1.2 (https://www.libsdl.org/projects/SDL_ttf/release-1.2.html)
Not sure what the so versions are...

Reply:
The other lib would be libSDL2_ttf it seems, so this isn't much of a problem.

Reply:
Issue for this is at https://github.com/haikuports/haikuports/issues/3878